uint/src/uint.rs

```diff
@@ -1281,14 +1264,25 @@ macro_rules! impl_std_for_uint {
 				s.parse().unwrap()
 			}
 		}
-	}
+
+		impl_heapsize_for_uint!($name);
+		// `$n_words * 8` because macro expects bytes and
+		// uints use 64 bit (8 byte) words
+		impl_quickcheck_arbitrary_for_uint!($name, ($n_words * 8));
+	);
 }
 
-#[cfg(not(feature="std"))]
+#[cfg(feature="std")]
 #[macro_export]
 #[doc(hidden)]
-macro_rules! impl_std_for_uint {
-	($name: ident, $n_words: tt) => {}
+macro_rules! impl_std_for_uint_internals {
```

Comment:
Why change the name of this macro? If it's used by another macro then you should use a private pattern of that macro rather than a `doc(hidden)` macro.